<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PatternSettingsWidget</class>
 <widget class="QWidget" name="PatternSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>425</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QLabel" name="label_2">
	 <property name="minimumSize">
      <size>
       <width>125</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>&amp;Foreground:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>m_fgColor</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="KColorButton" name="m_fgColor">
	 <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>m_fgColor</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>187</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_3">
	 <property name="minimumSize">
      <size>
       <width>125</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>&amp;Background:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>m_bgColor</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="KColorButton" name="m_bgColor">
	 <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>m_bgColor</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="ItemsView" name="m_pattern">
     <property name="movement">
      <enum>QListView::Static</enum>
     </property>
     <property name="flow">
      <enum>QListView::LeftToRight</enum>
     </property>
     <property name="isWrapping" stdset="0">
      <bool>true</bool>
     </property>
     <property name="resizeMode">
      <enum>QListView::Adjust</enum>
     </property>
     <property name="layoutMode">
      <enum>QListView::Batched</enum>
     </property>
     <property name="spacing">
      <number>2</number>
     </property>
     <property name="viewMode">
      <enum>QListView::IconMode</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ItemsView</class>
   <extends>QListView</extends>
   <header>itemsview.h</header>
  </customwidget>
  <customwidget>
   <class>KColorButton</class>
   <extends>QPushButton</extends>
   <header>kcolorbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
